core/system/inc/secure_gateway.h

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
👍 As soon as this PR is merged I'll create a new one that removes the secure gateway. This is to create a single point of removal that we can point to in the future if we want to.